akka-remote-tests/src/multi-jvm/scala/akka/remote/RemotingMultiNodeSpec.scala

```diff
@@ -24,7 +27,12 @@ object RemotingMultiNodeSpec {
 
 }
 
-abstract class RemotingMultiNodeSpec(config: MultiNodeConfig) extends MultiNodeSpec(config)
+abstract class RemotingMultiNodeSpec(config: MultiNodeConfig) extends MultiNodeSpec(config, config ⇒ {
+  val name = MultiNodeSpec.getCallerName(classOf[RemotingMultiNodeSpec])
+  //  When running multi-jvm tests with port = 0 two ActorSystems
+  //  can select the same port so retry once
+  Try(ActorSystem(name, config)).getOrElse(ActorSystem(name, config))
```

Comment:
could this be pushed down to `MultiNodeSpec`? because then all such tests benefit from this, e.g. all cluster tests

Reply:
The reason I didn't was that is public in the testkit where as Remoting is private to our tests. 

Happy to move if you think that's okay, WDYT @patriknw ?

Reply:
I think it's fine to add the feature to public, but perhaps we should limit it to retry only for a specific exception.

Reply:
Okay, will move

Reply:
hah i actually changed that to be a Runtime exception in case of Artery (before it didn't fail) as it is in the media driver. I'll make the Artery case a more specific exception then fix this